sdk/compute/Azure.ResourceManager.Compute/src/Generated/Models/OperatingSystemStateTypes.cs

```diff
@@ -7,12 +7,12 @@
 
 namespace Azure.ResourceManager.Compute.Models
 {
-    /// <summary> The OS State. </summary>
+    /// <summary> This property allows the user to specify whether the virtual machines created under this image are &apos;Generalized&apos; or &apos;Specialized&apos;. </summary>
     public enum OperatingSystemStateTypes
```

Comment:
Use a singular type name for an enumeration unless its values are bit fields. Use a plural type name for an enumeration with bit fields as values, also called flags enum, https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-classes-structs-and-interfaces#naming-enumerations